ES/ESdd044.xml

```diff
@@ -352,60 +357,98 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+                        </item>
+                        
+                        <item xml:id="e9">
+                           <locus target="#16vc"/>
+                           <desc type="Correction">Erasure marked with a crude black line.</desc>
+                        </item>
+                        
+                        <item xml:id="e10">
+                           <locus target="#37vc #44rb #55vc #60rb #62rb #62vc"/>
+                           <desc type="Correction">Immediate corrections indicated by thin frames in black ink around the word or the sentence.</desc> 
+                        </item>
+                        
+                        <item xml:id="e11">
+                           <locus target="#1r #1v"/>
+                           <desc>Pen trial.</desc>
+                        </item>
+                        
+                        <item xml:id="e12">
+                           <locus target="#138vc"/>
+                           <desc>Illegible note in pencil.</desc>
+                        </item>
                      </list>
                   </additions>
+                  
                   <bindingDesc>
                      <binding contemporary="false">
-                        <decoNote xml:id="b1">Two wooden boards covered with reddish-brown tooled
-                           leather; textile inlays.&#xD; Two pairs of sewing stations.</decoNote>
-                        <decoNote xml:id="b2" type="bindingMaterial">
+                        <decoNote xml:id="b1">Two wooden boards.</decoNote>
+                           <decoNote xml:id="b2" type="Cover">Reddish-brown tooled
+                              leather; <term key="tooling">blind tooled ornament on the cover and on the turn-ins</term> (8 tools were used: <term key="crissCross">criss-cross</term>, 
+                              <term key="corniForm">corni-form</term>, <term key="XForm">x-form</term>, cross hatching,
+                              double circle with dot, single line fillet, double sine wave with dots, double crescent with eye);
+                              <term key="textileInlay">textile inlays</term>.</decoNote> 
+                        <decoNote xml:id="b3" type="Sewing">Two pairs of sewing stations.</decoNote>
+                        <decoNote xml:id="b4" type="SewingStations">4</decoNote>
+                        <decoNote xml:id="b5" type="bindingMaterial">
                            <material key="wood"/>
                            <material key="leather"/>
                            <material key="textile"/>
                         </decoNote>
-                        <decoNote xml:id="b3" type="Endbands"/>
-                        <decoNote xml:id="b4" type="Other">Small holes are visible on the spine fold
+                        <decoNote xml:id="b6" type="Endbands">Missing</decoNote>
```

Comment:
I would say, if the endbands were never there, no indication is needed. If they are torn off, but you are sure they existed, you say this explicitely, smth like "Endbands (both are torn off)", and make a short statement also under "condition".